pkgs/by-name/mt/mtkclient/package.nix

```diff
@@ -0,0 +1,70 @@
+{
+  lib,
+  fetchFromGitHub,
+  makeWrapper,
+  python3,
+  stdenv,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "mtkclient";
+  version = "2.0.1";
+
+  nativeBuildInputs = [
+    makeWrapper
+  ];
+
+  strictDeps = true;
+
+  python = python3.withPackages (
+    ps: with ps; [
+      capstone
+      colorama
+      flake8
+      fusepy
+      keystone-engine
+      mock
+      pycryptodome
+      pycryptodomex
+      pyserial
+      pyside6
+      pyusb
+      setuptools
+      shiboken6
+      unicorn
+      wheel
+    ]
+  );
+
```

Comment:
```suggestion
  buildInputs = [ python ];
```

Reply:
Oh, that is how you make it automatically figure out the wrappers. Good to know, thanks!